central/detection/service/service_impl.go

```diff
@@ -226,16 +238,15 @@ func (s *serviceImpl) enrichAndDetect(ctx context.Context, enrichmentContext enr
 	}, nil
 }
 
-func (s *serviceImpl) runDeployTimeDetect(ctx context.Context, enrichmentContext enricher.EnrichmentContext, obj k8sRuntime.Object, policyCategories []string) (*apiV1.DeployDetectionResponse_Run, error) {
+func (s *serviceImpl) convertK8sResource(obj k8sRuntime.Object) (*storage.Deployment, error) {
```

Comment:
I think, the `s` is not used here
```suggestion
func convertK8sResource(obj k8sRuntime.Object) (*storage.Deployment, error) {
```